System/System.h

```diff
@@ -56,6 +56,19 @@ namespace RTE {
 		static bool MakeDirectory(const std::string &pathToMake);
 #pragma endregion
 
+#pragma region Filesystem
+		/// <summary>
```

Comment:
Fix comment formatting.

Reply:
not entirely sure what the problem is here

Reply:
The param entry is problematic
1. Your comment says filename but your param is named pathToCheck
2. The description of the param should be on one line, and should end in a period.
3. You're missing a return entry for the method comment

Also, EnableCaseSensitivity is missing a param, a return, and a period at the end of the description.

Also also, IsCaseSensitveEnabled is misspelled and the <returns> param in it is misspelled.

Also also also, you've got a typos of `wether` that should be `whether` on several of these.